### DISCUSSION TITLE
Is it possible to uninstall k3s, but keep docker images?

### DISCUSSION THREAD

**Discussion by SpringHgui** [k3s-io/k3s#8037] [General]
**Is your feature request related to a problem? Please describe.**

I just want to reinstall k3s，but all my image caches are deleted, and it takes a lot of time to download the images

**Describe the solution you'd like**
`/usr/local/bin/k3s-uninstall.sh` Supports parameter passing

**Comment by brandond** [comment]
No, the uninstall uninstalls everything; we don't have any plans to allow for partial uninstalls. If you just want to reset K3s, you can run the killall script, and delete /var/lib/rancher/k3s/server. Another option would be to move `/var/lib/rancher/k3s/agent/containerd` somewhere before uninstalling, and then move it back before reinstalling K3s, although a better idea would probably be to set up a pull-through registry mirror to cache images.
